tools/test.py

```diff
@@ -1265,6 +1265,9 @@ def BuildOptions():
   result.add_option("--no-store-unexpected-output", 
       help="Deletes the temporary JS files from tests that fails",
       dest="store_unexpected_output", action="store_false")
+  result.add_option("-r", "--run",
+      help="Run test n modulo m (--run=n,m with n < m)",
```

Comment:
`n modulo m` is not exactly reflects the intention (at leazst as per the `xrange`). "Run every mth test starting from n" would be better I guess.